src/app/services/cinchy-query.service.ts

```diff
@@ -87,7 +88,26 @@ export class CinchyQueryService {
     const params = {
       '@formId': formId ? formId : sessionStorage.getItem('formId')
     };
-    return this.cincyService.executeQuery('Cinchy Forms', 'Get Form MetaData', params);
+    return this.cincyService.executeQuery('Cinchy Forms', 'Get Form MetaDataNew', params);
```

Comment:
I would rename this uqery back to Get Form MetaData. The "New" portion was just for your testing purposes

Reply:
Resolved